### ISSUE TITLE
Additional Files not found when running pdflatex in PDFExporter

### ISSUE THREAD

**Issue by stonefly** [jupyter/nbconvert#52]
I recently encounter a problem that does not happen when I use ipython 2.x.

I convert ipynb to pdf using my own template which refers to additional files like logo.eps etc.

Before I always create my own working directory and put all files (ipynb file, tplx file, eps file, etc) under it and run nbconvert. It works fine.

However it breaks recently due to that in the new PDFExporter, the pdflatex runs in another temporary directory where only the tex files and the output files along with the latex generations are written to that directory. So I will get a '***' File Not Found error finally because the pdflatex cannot find the 'eps' file.

In contrast, in the old days, the 'PDF' postprocessor (deprecated now) did run in-place so there was no such problems.

Do we have an elegant way to specify additional files that should run with pdflatex? I found it's not obvious to insert the files i need in `resources['outputs']`, although it's not impossible

![nbconvert_bug](https://cloud.githubusercontent.com/assets/689302/8360089/563a2fa4-1b62-11e5-9437-f680114ff82a.JPG)

**Comment by takluyver** [comment]
If you're customising PDF creation that much, it may be simpler to use `--to latex` and then build the PDF yourself using `pdflatex`.

**Comment by stonefly** [comment]
Yes, I use this way to fix it currently. 

I did not do many customizations, I only use a customized template and a customized logo. 

Any reasons that we have to use another temporary folder here? It breaks something that used to work.

I think the users should expect --to=pdf should be equivlent to --to=latex and then run pdflatex 3 times. But now they will behave differently.

**Comment by takluyver** [comment]
If you just want the PDF, you shouldn't get additional files written in your working directory by the implementation of how the PDF is generated. It's encapsulation - intermediate files are written somewhere where you don't see them. If you do want to modify things at that stage, e.g. to add another file at the latex level, it involves an extra step.

**Comment by stonefly** [comment]
I agree that it's better to run pdflatex in a temporary directory to the oldways generate intermidiate files and then clean them. It's quite annoying that even you run pdflatex with -output-directory, it still generates some files in the current folder in some cases, quite annoying.

I am just thinking if i could specify files that I would like to have along with pdflatex in the PDFExport config? So that these files will be copied to the new temp directory.

Do you think this makes sense?

**Comment by takluyver** [comment]
That sounds reasonable, though I haven't thought about it in much detail. Do you want to make a PR?

**Comment by stonefly** [comment]
Sure, let me try.

Many thanks.
